plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -87,6 +87,9 @@ var (
 
 const (
 	DefaultProvider = "DefaultProvider"
+	MaxUint         = ^uint(0)
```

Comment:
can we externalized these or add them to `math.MaxInt` `math.MaxUInt` ?  I'm pretty sure we can, and If not, we should add them to a util somewhere.